radix-engine/src/blueprints/resource/auth_zone.rs

```diff
@@ -31,8 +31,11 @@ impl AuthZoneBlueprint {
             RuntimeError::SystemUpstreamError(SystemUpstreamError::InputDecodeError(e))
         })?;
 
-        let auth_zone_handle =
-            api.actor_lock_field(AuthZoneOffset::AuthZone.into(), LockFlags::MUTABLE)?;
+        let auth_zone_handle = api.actor_lock_field(
+            OBJECT_HANDLE_SELF,
```

Comment:
I wonder if we should add `actor_lock_self_field` etc to the API to make it slightly nicer to use, so we don't have to add `OBJECT_HANDLE_SELF` everywhere :)